### ISSUE TITLE
training.py Running Time

### ISSUE THREAD

**Issue by tomorrow1pan** [PatrickLib/captcha_recognition#23]
I used Tensorflow-CPU. The captcha_train.py had run over 3 hours and the step was over 2700, but the loss was between 2.9 and 3. It decreased very slowly. Can you tell me how long did your training.py  run? And, doed it stop until the loss reach zero ?  THX

**Comment by He-haitao** [comment]
i face the same problem

**Comment by tomorrow1pan** [comment]
> i face the same problem

**Comment by tomorrow1pan** [comment]
After trying several times, I had found that you needn't to wait the train.py to converge and stop. The trained models was saved under folder _captcha_train_. And you can run eval.py and test.py at any time, no matter the trainig.py is runnnig or stop. But the training steps will affect the prediction accuracy. You can choose the best trained model. Hope this can help you. @He-haitao

**Comment by He-haitao** [comment]
Yeah, i know this. but when i run captcha_eval.py, the accuracy is 0.00. my step is about 7000-8000steps and from 2000steps ,the loss is around 3.9, i just saw that they run 20000steps and get a pretty good result, i just don't know why.

**Comment by He-haitao** [comment]
do you get any result when you run the eval.py?

**Comment by tomorrow1pan** [comment]
yeah, The accuracy rate of prediction and evaluation was around 45% after 16000 steps training. But at the beginning of training, the result of eval.py and recognize.py was 0%. I  didn't know why. Maybe the train.py had to run at least certain steps??? @He-haitao
